src/content/en/2020/jamstack.md

```diff
@@ -3,21 +3,265 @@
+
+The [HTTP Archive][1] crawls [millions of pages][2] every month and runs them through a private instance of [WebPageTest][3] to store key information on every page. (You can learn more about this in our [methodology][4]). In the context of Jamstack, HTTP Archive provides extensive information on the usage of the frameworks and CDNs for the entire web. This chapter consolidates and analyzes many of these trends.
+
+### Goals: Growth of Jamstack & Real User Perf
+
+The goals of this chapter are to estimate and analyze the growth of the Jamstack sites, the performance of popular Jamstack frameworks, as well as an analysis of real user experience using the Core Web Vitals metadata.
+
+## Adoption: What is the overall level of Jamstack adoption amongst all websites, across desktop and mobile?
+
+Our analysis throughout this work looks at desktop and mobile websites. The vast majority of URLs we looked at are in both datasets, but some URLs are only accessed by desktop or mobile devices. This can cause small divergences in the data, and we thus look at desktop and mobile results separately.
+
+{{ figure_markup(
+  image="jamstack-adoption.png",
+  caption="Jamstack adoption trend.",
+  description="Bar chart showing the increase in the level of Jamstack adoption in 2019 and 2020. Mobile has increased from 0.40% to 0.72%. Desktop has increased from 0.27% to 0.68%.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ-zBygubF5NuAXOGScVLhnozB_6Pi_0z4B57cP_I6AfeBuZCGzs3aP_1nPjPp37aNRATRYC6EfOeYX/pubchart?oid=1650360073&format=interactive",
+  sheets_gid="908645975",
+  sql_file="ssg_compared_to_2019.sql"
+  )
+}}
+
+More than 0.7% of web pages are powered by Jamstack, an increase of over 79% from 2019. This breaks down to 0.72% on desktop, up from 0.40%  in 2019, and 0.68% on mobile, up from 0.27% in 2019. We believe this is a sign of the steady growth of the Jamstack community. You can read more about ours on the [Methodology][4] page.
+
+<figure markdown>
+
+|   Year   | Desktop | Mobile |
+| -------- | ------- | ------ |
+| 2019     | 0.40%   | 0.27%  |
+| 2020     | 0.72%   | 0.68%  |
+| % Change | 79%     | 154%   |
+
+<figcaption>{{ figure_link(caption="Jamstack adoption statistics.", sheets_gid="908645975", sql_file="ssg_compared_to_2019.sql") }}</figcaption>
+</figure>
+
+The increase in desktop web pages powered by a Jamstack framework is 79% from last year. On mobile, this increase is almost two folds, at 154%. This is a significant growth from 2019, especially for mobile pages.
+
+## Top SSGs YoY: Which are the most common Jamstack frameworks across desktop and mobile sites?
+
+Our analysis counted 12 separate Jamstack frameworks. Only five frameworks had more than 1% share. Next.js, Gatsby, Hugo, Jekyll being the top contenders for the Jamstack market share.
```

Comment:
I think we need to talk about "the Eleventy problem" here. Or in the introduction.

```suggestion
Our analysis counted 12 separate Jamstack frameworks. Only five frameworks had more than 1% share. Next.js, Gatsby, Hugo, Jekyll being the top contenders for the Jamstack market share.

<p class="note">It should be noted that our analysis is limited by those Jamstacks that make themselves easily identified using [Wappalyzer](./methodology#wappalyzer). This means our data does not include some popular Jamstacks like [Eleventy](https://github.com/11ty/eleventy/) which [make a deliberate choice to not make themselves identifiable](https://twitter.com/eleven_ty/status/1334225624110608387?s=20). While we would ideally include all Jamstacks, we believe there is still value in analysing what we have</p>
```